downloadservice/app.py

```diff
@@ -63,6 +70,44 @@ def urljoin_multipart(*args):
 default_chunk_size = 10 * 1024 * 1024
 
 
+def verify_certificate(cabundle, certificate):
```

Comment:
I'd start moving this to another file - it's a bit specialized functionality and the file is big already